File: blocks/types/sand.go
package blocks

// SandBlock is a block of sand, easy to destroy/collect by
// hand
type SandBlock struct {
    Block
    lifespan, health f32
}

// NewSandBlock returns a new SandBlock
fn NewSandBlock() *SandBlock {
    return &SandBlock{
        lifespan: 5,
        health: 5,
    }
}

File: src/blocks/types/sand.rs
/// SandBlock is a block of sand, easy to destroy/collect by
/// hand
struct SandBlock {
    lifespan: f32, 
    health: f32,
}

impl SandBlock {
    /// Returns a new SandBlock
    fn new() -> Self {
        Self {
            lifespan: 5,
            health: 5,
        }
    }
}

impl Block for SandBlock {

}
